core/app/assets/stylesheets/refinery/site_bar.css.scss

```diff
@@ -1,85 +1,137 @@
 @import '_mixins';
 
-#site_bar_content {
-  height:47px;
-  position: relative;
-  margin: 0px auto 10px auto;
-  text-align: left;
-  width: 1000px;
-}
+$standard_rounding: 0px;
```

Comment:
What about to move vars to some separate scss file to make them easy overridable?